docs/ui/sso.md

```diff
@@ -0,0 +1,49 @@
+---
+description: Configure single sign-on (SSO) for your Prefect Cloud users.
+tags:
+    - UI
+    - dashboard
+    - Prefect Cloud
+    - enterprise
+    - teams
+    - workspaces
+    - organizations
+    - single sign-on
+    - SSO
+    - authentication
+---
+
+# Single Sign-on (SSO) <span class="badge cloud"></span> <span class="badge orgs"></span> <span class="badge enterprise"></span>
+
+Prefect Cloud [Enterprise plans](https://www.prefect.io/pricing) offer single sign-on (SSO) integration with your team’s identity provider. SSO integration can bet set up with any identity provider that supports: 
+
+- OIDC
+- SAML 2.0
+
+When using SSO, Prefect Cloud won't store passwords for any accounts managed by your identity provider. Members of your Prefect Cloud organization will instead log in to the organization and authenticate using your identity provider.
+
+When SSO integration has been set up, all other authentication methods are disabled, so that you can ensure that authentication is managed exclusively by your identity provider.
```

Comment:
```suggestion
Once your SSO integration has been set up, non-admins will be required to authenticate through SSO when accessing organization resources.
```

For clarity - other authentication methods are not disabled but authenticating with an organization's SSO provider is necessary to access any organization-level resources.